3회차/문현동/1_소득불균형.py

```diff
@@ -1,3 +1,17 @@
-import sys
+# import sys
 
-sys.stdin = open("_소득불균형.txt")
+# sys.stdin = open("_소득불균형.txt")
+
+T = int(input())
+
+for test_case in range(1, T + 1):
+    N = int(input())
+    income_list = list(map(int, input().split()))
+    
+    low_income_list = []
+    
+    for workerS_income in income_list:
+        if workerS_income <= sum(income_list) / N: # 평균을 낸 다음 어떤 일꾼의 수입이 그 평균보다 적으면
+            low_income_list.append(workerS_income) # 그 작은 사람들을 '적은 수입 리스트' 에 추가
```

Comment:
`low_income_list` 리스트에 가능한 수입을 추가해 길이를 확인해주셨네요.
정답을 저장할 변수를 선언하고 갱신해주는 방법도 좋을 것 같아요!
```suggestion
    low_income = 0
    
    for workerS_income in income_list:
        if workerS_income <= sum(income_list) / N: # 평균을 낸 다음 어떤 일꾼의 수입이 그 평균보다 적으면
            low_income += 1
```